editors/linux-wps-office/Makefile

```diff
@@ -0,0 +1,55 @@
+PORTNAME=	wps-office
+DISTVERSION=	11.1.0.10161
+CATEGORIES=	editors linux
+MASTER_SITES=	https://wdl1.pcfg.cache.wpscdn.com/wpsdl/wpsoffice/download/linux/10161/
+PKGNAMEPREFIX=	linux-
+DISTNAME=	${PORTNAME}-${PORTVERSION}.XA-1.x86_64
+
+MAINTAINER=	ygy@FreeBSD.org
+COMMENT=	Kingsoft WPS Office free for Linux
+
+LICENSE=	commercial
+LICENSE_NAME=	End User License Agreement
+LICENSE_TEXT=	${RESTRICTED}
+LICENSE_PERMS=	no-auto-accept no-dist-mirror no-dist-sell no-pkg-mirror no-pkg-sell
+
+RESTRICTED=	Source recipient must acknowledge the license. \
+		Reproduction or redistribution prohibited. \
+		See https://www.wps.com/eula.
+
+USES=		desktop-file-utils linux shebangfix
+INSTALLS_ICONS=	yes
+USE_LINUX=	png pulseaudio-libs xorglibs
+
+SHEBANG_FILES= 	usr/bin/et \
+		usr/bin/wps \
+		usr/bin/wpspdf \
+		usr/bin/wpp
```

Comment:
sorting.

Reply:
This sorting looks fine by me?

Reply:
I think usr/bin/wpp should be placed before usr/bin/wps ?

Reply:
Oh right.............. So obvious. Need coffee. /facepalm